src/opensynth/__init__.py

```diff
@@ -0,0 +1,4 @@
+# Copyright Contributors to the Opensynth-energy Project.
+# SPDX-License-Identifier: Apache-2.0
+
+__version__ = "0.0.1"
```

Comment:
Outdated - next commit fixes this to __version__="0.0.0-alpha".

When creating release branch that ultimately publish this to PyPI - we'll bump this up in the release PR.